include/avif/avif.h

```diff
@@ -336,7 +336,7 @@ enum
     // This is actually reserved, but libavif uses it as a sentinel value.
     AVIF_TRANSFER_CHARACTERISTICS_UNKNOWN = 0,
 
-    AVIF_TRANSFER_CHARACTERISTICS_BT709 = 1,
+    AVIF_TRANSFER_CHARACTERISTICS_BT709 = 1, // Same as sRGB
```

Comment:
remove this comment, the sRGB transfer curve is actually AVIF_TRANSFER_CHARACTERISTICS_SRGB
only the primaries match BT709